new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/ActionConstants.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes.actions;
+
+public interface ActionConstants {
+
+  // Work directory for the integration test suite
+  public static final String WORK_DIR 
+      = System.getProperty("java.io.tmpdir") + "/it-results";
```

Comment:
needs a null check

Reply:
I assumed this system property would never be null. I don't know what to do if it actually returns null. 
I am using this as a temporary solution for a to-be-added script that runs upfront to setup necessary environment for the integration test suite. That script would set the WORK_DIR to an appropriate directory according to the platform where the integration test runs. Once we have that script in place, we don't need this anymore.

Reply:
I am okay without having a null check here.  If this fails, the build should fail.  Because this is in the test code, the NPE stacktrace is sufficient to debug it, in my opinion.  Shouldn't this "/" be File.separator also?

Reply:
java.io.tmpdir is guaranteed to have a value by the System.getProperty javadoc.

Reply:
ok - ignore me :)